db/migrations/20190314-fix-01.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE acos_beneficiaries (
+  aco_id uuid NOT NULL,
+  beneficiary_id int NOT NULL
+);
+
```

Comment:
Don't you want some constraints on this table? i.e., I'm assuming you don't want to allow duplicate (aco_id, beneficiary_id) tuples? and what about foreign keys constraints? maybe that's elsewhere?

Reply:
I think it is here.  I remembered seeing it in the initial code review
https://github.com/CMSgov/bcda-app/blob/cd413cc3abbdfc167c74e3cc47ba52db045d6493/bcda/models/models.go#L33

Reply:
Yep, what Knoll said. It's relying on GORM to add those constraints.